test/cli/simple-package/bar/bar.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
+
 # typed: strict
 
 module Project::Bar
   class BarClass
     extend T::Sig
-    sig {params(value: Integer).void}
+    sig { params(value: Integer).void }
     def initialize(value)
       @value = T.let(value, Integer)
     end
   end
+
+  class UnexportedClass; end
```

Comment:
Adding an additional constant to ensure that un-exported constants don't get exposed by mistake.